index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <main id="main">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+<h1 id="title">"Kaneki Kun"</h1>
+  <body>
+    "Hello World, we are software developers!"
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <div id="img-div">
+      <img id="image" src="https://cdn.pixabay.com/photo/2018/10/14/00/15/ken-3745445_960_720.jpg" alt="Image of kaneki">
```

Comment:
Add this image inside an anchor tag to respect best practices for HTML and CSS. Check this link: https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md

Reply:
- [ ] I suggest that you update the link in the same way as i suggested in the comment attached previously. This will help us to make the review shorter and readable ☺️ .

Reply:
Add this image inside an anchor tag to respect best practices for HTML and CSS. Check this [link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)